internal/pkg/term/selector/selector.go

```diff
@@ -199,6 +205,13 @@ type AppPipelineSelect struct {
 	*CodePipelineSelect
 }
 
+// PipelineEnvsConnectionSelect is a selector for pipeline envs and CSC.
+type PipelineEnvsConnectionSelect struct {
```

Comment:
```suggestion
type ConnectionSelect struct {
```

Reply:
Previously, the `PipelineEnvSelector`, used only in `pipeline_init`, was the selector. Now that it's also implementing `Connection`, I thought I would combine the two with a name that reflects that, like some of the other selectors. Is it better to separate them?